### PR TITLE
Awesome Books project

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello!
please find below some interesting javascript features we just set up in this project.
kindly have a look and let us know if there's anything that needs improvements.

### General requirements

1. Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config). 💯 
2. Make sure that you used correct [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). 💯 
3. Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). 💯 

Faithfully yours

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @team 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't follow the [README template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+document.getElementById('form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const randomId = Math.floor(Math.random() * 100);
+  const element = {
+    id: randomId,
+    title: document.getElementById('title').value,
+    author: document.getElementById('author').value,
+  };
+  books.push(element);
```
Comment (omar-muhamad) [inline js/main.js:39]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20add%20a%20new%20book%20to%20the%20collection%2C%20with%20title%20and%20author.), Please a separate function to add a book to book collection and you can name it `addBook` then you can call it here inside the submit event listener.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  element.appendChild(document.createTextNode(book.author));
+  div.appendChild(element);
+  element = document.createElement('button');
+  element.setAttribute('type', 'button');
+  element.appendChild(document.createTextNode('Remove'));
+  element.addEventListener('click', () => {
+    // remove the book from the screen
+    div.remove(book.id);
+    // remove the book from the local storage
+    const newArray = books.filter((book2) => book2.id !== book.id);
```
Comment (omar-muhamad) [inline js/main.js]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).), Please create a separate function to remove a book from book collection using [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method then you can call it here in `showBook` function.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by chiztechnology** [comment]
thank you dear @Omar-Muhamad .
we've updated the content and fixed the error. 🙏 👏 

best regards

**Review by Santiago220991 — APPROVED** [review]
Hi @chiztechnology @chiztechnology, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**